src/main/scala/mesosphere/marathon/MarathonSchedulerService.scala

```diff
@@ -76,6 +75,7 @@ class MarathonSchedulerService @Inject() (
 
   val log = Logger.getLogger(getClass.getName)
 
+  // FIMXE: Remove from this class
```

Comment:
I thinks it is amusing that I made a type there ;)

No, no left over. I would like to fix that eventually but not in this patch.